packages/zwave-js/src/lib/commandclass/BarrierOperatorCC.ts

```diff
@@ -81,22 +81,22 @@ export class BarrierOperatorCCSet extends BarrierOperatorCC {
 				ZWaveErrorCodes.Deserialization_NotImplemented,
 			);
 		} else {
-			this.targetValue = options.targetValue;
+			this.state = options.state;
 		}
 	}
 
-	public targetValue: boolean;
+	public state: State.Open | State.Closed;
 
 	public serialize(): Buffer {
-		const payload: number[] = [this.targetValue ? 0xff : 0x00];
+		const payload: number[] = [this.state ? 0xff : 0x00];
```

Comment:
`this.state` is already either `0xff` or `0x00`, so you can just do `this.payload = Buffer.from([this.state]);`